src/components/MyAccountContent/MyAccountContent.js

```diff
@@ -22,7 +22,7 @@ class MyAccountContent extends Component {
           <WrapStyled>{children}</WrapStyled>
         ) : (
           <Scrollbars
-            style={{ flexGrow: '1', width: 'unset' }}
+            style={{ height: '100%', flexGrow: '1', width: 'unset' }}
```

Comment:
Maybe we could extract styles to separate file or use styled component instead.

Reply:
in this particular case, I would leave it as it is, as this is a recommended way for Scrollbars to style